tests/virtualenv_tools_test.py

```diff
@@ -76,28 +76,29 @@ def test_each_part_idempotent(tmpdir, venv, capsys):
     assert out == expected
 
 
-def _assert_activated_sys_executable(path):
+def _assert_activated_sys_executable(path, py):
     exe = subprocess.check_output((
         'bash', '-c',
-        ". {} && python -c 'import sys; print(sys.executable)'".format(
-            pipes.quote(path.join('bin/activate').strpath),
+        ". {activate} && {py} -c 'import sys; print(sys.executable)'".format(
+            activate=pipes.quote(path.join('bin/activate').strpath),
+            py=py
         )
     )).decode('UTF-8').strip()
-    assert exe == path.join('bin/python').strpath
+    assert exe == path.join('bin/{py}'.format(py=py)).strpath
```

Comment:
the parts where you take pains to change "python" to sometimes-pypy don't feel right
"python" should simply map to pypy in the pypy case
thats >50% of the changes as far as i see